Dockerfile.nfer

```diff
@@ -0,0 +1,40 @@
+# Use the specified base image
+FROM hz-registry.nferx.com/ubuntu as builder
```

Comment:
@vilay-nference Where is the dockerfile for this image?  It is not immediately accessible, so we cannot verify its contents.  

In general, we use base images provided directly from first-party sources (e.g. `https://hub.docker.com/_/ubuntu`).  

Please change this to use a standard base image and then we can re-evaluate.

Reply:
This is used for internal purpose. I'll delete this file .. not required for gatk